### PR TITLE
data: hong-codes

### PR DESCRIPTION
**Pull request by hong-codes**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2831"></a>

### CODE REVIEW

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please remove the comma at the end of line 27

File: data/hong-codes.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/hongcodes/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Follow me on Twitch",
+      "url": "https://www.twitch.tv/hongcodes",
+      "icon": "FaTwitch"
+    },
```
Comment (eddiejaoude) [inline data/hong-codes.json:27]: Looks good, but this trailing comma `,` is not needed

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Comment by hong-codes** [comment]
Thank you for the review. New to collaborating on GitHub, so still figuring my way around.

File: data/hong-codes.json (hunk 1 above)
Comment (hong-codes) [reply data/hong-codes.json]: Thank you, I updated it now.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**